cluster/libvirt-coreos/util.sh

```diff
@@ -51,6 +52,19 @@ function detect-nodes {
   KUBE_NODE_IP_ADDRESSES=("${NODE_IPS[@]}")
 }
 
+function set_service_accounts {
+    SERVICE_ACCOUNT_KEY=${SERVICE_ACCOUNT_KEY:-"/tmp/kube-serviceaccount.key"}
+    # Generate ServiceAccount key if needed
+    if [[ ! -f "${SERVICE_ACCOUNT_KEY}" ]]; then
+      mkdir -p "$(dirname ${SERVICE_ACCOUNT_KEY})"
+      openssl genrsa -out "${SERVICE_ACCOUNT_KEY}" 2048 2>/dev/null
+    fi
+
+    mkdir -p  "$POOL_PATH/kubernetes/certs"
```

Comment:
In `libvirt-coreos` setup, the `$POOL_PATH/kubernetes` directory of the host (machine where `kube-up.sh` is launched) is mounted on each VM in `/opt/kubernetes`.

It is the most efficient way to push things to the VMs.

I’m fine with that.